builtin/credential/cert/path_login.go

```diff
@@ -371,6 +382,33 @@ func (b *backend) checkForValidChain(chains [][]*x509.Certificate) bool {
 	return false
 }
 
+func (b *backend) checkCIDR(cert *CertEntry, req *logical.Request) error {
+
+	if len(cert.BoundCIDRs) <= 0 {
```

Comment:
Len is never going to be below zero.  I think `==` makes more sense here.

Reply:
That's fine! A previous dev team I was on preferred it that way because it was "more defensive". Will update.